content/02.main-text.md

```diff
@@ -143,6 +144,25 @@ Styles define how references are constructed from bibliographic metadata, contro
 Thousands of journals have [predefined styles](http://editor.citationstyles.org/searchByName/).
 As a result, adopting the specific bibliographic format required by a journal usually just requires specifying the style's source URL in the Manubot configuration.
 
+To make citation-by-identifier easily usable outside of Manubot manuscripts, we created the `manubot cite` command line utility, available as a [Python package](https://pypi.org/project/manubot/).
+This utility produces CSL Items (i.e. JSON-formatted reference metadata) for a list of citations.
+For example, `manubot cite doi:10.1098/rsif.2017.0387`, returns the CSL Item for the Deep Review.
+In addition, the utility can render the references into a formatted bibliography.
+For example, the following command outputs a Markdown reference list according to the bibliographic style of _PeerJ_:
+
+```sh
+manubot cite --render --format=markdown \
+  --csl=https://github.com/citation-style-language/styles/raw/master/peerj.csl \
+  pmid:29618526 doi:10.1038/550143a
+```
+
+If the user does not specify a CSL style, the program defaults to using the Manubot style, which focuses on information relevant for modern scholarly workflows and omits the rest.
```

Comment:
This paragraph reads more like documentation rather than information that will get readers excited about Manubot and help them understand what it contributes to open collaborative writing.

Reply:
Agree, especially about the CSL. Having a default CSL is probably not part of the core message.

Reply:
P.S. I also wanted to cite my blog post on numeric versus author citation styles. This was a bit of original research that was largely motivated by Manubot... I wanted to get to the bottom of whether we should use numeric or author-style, and what the differences were.